public/pages/AnomalyCharts/containers/AnomalyHeatmapChart.tsx

```diff
@@ -109,28 +125,57 @@ export const AnomalyHeatmapChart = React.memo(
       });
 
       return [
-        COMBINED_OPTIONS,
+        getViewableCombinedOptions(
+          COMBINED_OPTIONS,
+          props.heatmapDisplayOption?.entityOption
+        ),
         {
           label: 'Individual entities',
-          options: individualEntityOptions,
+          options: individualEntityOptions.reverse(),
```

Comment:
Input of the method `inputHeatmapData` has entities in ascending order by sorting field. And then the `individualEntityOptions` array is in ascending order, but we want to list those options in combo box in descending because heatmap is showing data in descending order. 

The reason why we make `inputHeatmapData` in ascending order, to make sure heatmap shows data in descending order, is because in plotly library, the origin of chart is at left bottom. If we make `heatmapData` in descending order, the biggest item will be shown on bottom, while smallest on top, but this looks ascending.